usage/camera/raspicam/timelapse.md

```diff
@@ -38,45 +40,30 @@ Make sure that you use e.g. `%04d` to make `raspistill` output each image to a n
 
 ## Stitching images together
 
-Now you'll need to stitch the photos together into a video. You can do this on the Pi using `mencoder` but the processing will be slow. You may prefer to transfer the image files to your desktop computer or laptop and produce the video there.
-
-Navigate to the folder containing all your images and list the file names in to a text file. For example:
+Now you'll need to stitch the photos together into a video. You can do this on the Pi using `ffmpeg` but the processing will be slow. You may prefer to transfer the image files to your desktop computer or laptop and produce the video there.
 
-```
-ls *.jpg > stills.txt
-```
-### On the Raspberry Pi
+First you will need to install `ffmpeg`, you may also need to update and upgrade your Pi to successfully install.
 
-Although it will be slow (due to encoding in software rather than using the Raspberry Pi hardware acceleration), you can stitch your JPEG images together using various available tools. This documentation will use `avconv`, which needs to be installed.
-```
-sudo apt install libav-tools
 ```
-Now you can use the tools to convert your JPEG files in to an H264 video file:
+sudo apt update
+sudo apt full-upgrade
+sudo apt install ffmpeg
```

Comment:
@XECDesign Should we be recommending users to do a `sudo apt full-upgrade` before doing `sudo apt install ffmpeg` ?

Reply:
I removed this bit. I initially tested with Raspberry Pi lite which required this, testing with a fresh copy of Raspberry Pi OS desktop, `ffmpeg` was actually already installed so update and upgrade was not required